scrapers/PlumperPass.yml

```diff
@@ -7,19 +7,15 @@ sceneByURL:
 xPathScrapers:
   sceneScraper:
     scene:
-      Title: 
-        selector: //div[@class="clearfix"]/h1/span/text()
-        postProcess:
-          - replace:
-              - regex: \"
-                with:
-      Details: //div[@class="movie-desc"]/text()
+      Title:
+        selector: //h2[@class="vidtitle"]
+      Details: //div[@class="vidinfo col-sm-12 col-md-12 col-lg-12 col-xl-3"]/p
       Tags:
         Name:
-          selector: //div[@class="tags clearfix"]/a/text()
+          selector: //p[@class="tags clearfix"]/a
       # Note about performers: The site only list 1 model even there is 2 in the video.
       Performers:
-        Name: //div[@class="clearfix"]/h1/a[1]/text()
+        Name: //div[@class="col-sm-12 col-md-12 col-lg-12 col-xl-9"]//a/text()
```

Comment:
`$h3: //div[@class="titlerow2"]//h3[@class="releases"]` and use
`$h3/a`  for performers 
`$h3/text()` date
?

Reply:
Yup that works as well :-)